airbyte-commons-worker/src/main/java/io/airbyte/workers/process/AirbyteIntegrationLauncher.java

```diff
@@ -212,7 +212,8 @@ private Map<String, String> getWorkerMetadata() {
         WorkerEnvConstants.WORKER_CONNECTOR_IMAGE, imageName,
         WorkerEnvConstants.WORKER_JOB_ID, jobId,
         WorkerEnvConstants.WORKER_JOB_ATTEMPT, String.valueOf(attempt),
-        EnvVariableFeatureFlags.USE_STREAM_CAPABLE_STATE, String.valueOf(featureFlags.useStreamCapableState()));
+        EnvVariableFeatureFlags.USE_STREAM_CAPABLE_STATE, String.valueOf(featureFlags.useStreamCapableState()),
+        EnvVariableFeatureFlags.AUTO_DETECT_SCHEMA, String.valueOf(featureFlags.autoDetectSchema()));
```

Comment:
This is tied to the comment about the orchestrator. I don't think we need this flag in the connectors, so we should not need to add it in `AsyncOrchestratorPodProcess`, `OrchestratorConstants` or `AirbyteIntegrationLauncher`. I think those occurrences of AUTO_DETECT_SCHEMA can be removed.